packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -189,6 +191,47 @@ export class PlaygroundWorkerEndpoint extends PHPWorker {
 	}
 }
 
+async function downloadWordPressAssets(php: PHP) {
+	const wpVersion = await getWordPressVersionFromPhp(php);
+	fetch(`${wordPressSiteUrl}/wp-${wpVersion}/wordpress-static.zip`)
```

Comment:
If we are going to use the async/await syntax here, why not also use if for the `fetch()`?

It also turns out that it is not actually necessary to wait for `response.body` once you have `response`. It's already a ReadableStream, not a Promise for one.

[An MDN example](https://developer.mozilla.org/en-US/docs/Web/API/Response/body#examples) does use `.then((response) => response.body)`, but it is unnecessary and is just mapping the `response` to `response.body`.

Reply:
If I understand correctly this is what you recommended [2e3910c](https://github.com/WordPress/wordpress-playground/pull/1532/commits/2e3910c074cf2320e9f89a09ca9f85e75d26cbc3)

Reply:
> If I understand correctly this is what you recommended [2e3910c](https://github.com/WordPress/wordpress-playground/pull/1532/commits/2e3910c074cf2320e9f89a09ca9f85e75d26cbc3)

Yes, that was it. It looks great to me. My comment was actually two separate comments, and you addressed both. I'm sorry if the combination made it unclear.